addons/website_sale_stock/models/website.py

```diff
@@ -8,26 +8,5 @@ class Website(models.Model):
 
     warehouse_id = fields.Many2one('stock.warehouse', string='Warehouse')
 
-    def _prepare_sale_order_values(self, partner_sudo):
-        values = super()._prepare_sale_order_values(partner_sudo)
-
-        warehouse_id = self._get_warehouse_available()
-        if warehouse_id:
-            values['warehouse_id'] = warehouse_id
```

Comment:
Out of curiosity: was this actually useful before your changes? I believe this value was overridden by `_compute_warehouse_id` anyway.

Reply:
I thought like that too, have not tested though.